src/main/java/de/csbdresden/stardist/StarDist2D.java

```diff
@@ -400,4 +411,59 @@ public static void main(final String... args) throws Exception {
         ij.command().run(StarDist2D.class, true, params);
     }
 
+    private void refresh() {
+        final HashMap<String, Object> params = new HashMap<>();
+
+        params.put("input", input);
+        params.put("modelChoice", modelChoice);
+        params.put("normalizeInput", true);
+        params.put("percentileBottom", percentileBottom);
+        params.put("percentileTop", percentileTop);
+        params.put("probThresh", probThresh);
+        params.put("nmsThresh", nmsThresh);
+        params.put("outputType", "ROI Manager");
+        params.put("nTiles", 1);
+        params.put("excludeBoundary", 2);
+        params.put("roiPosition", Opt.ROI_POSITION_AUTO);
```

Comment:
Why did you hardcode some parameters (e.g. `normalizeInput`)?

Reply:
Sorry, it did not use the actual parameters from the GUI. It should be resolved with the recent changes in  fce718b.

https://github.com/stardist/stardist-imagej/blob/fce718b39437e26022283510d47bae87ba6da589/src/main/java/de/csbdresden/stardist/StarDist2D.java#L417-L427